packages/babel-plugin-relay/BabelPluginRelay.js

```diff
@@ -17,6 +17,12 @@ const getValidGraphQLTag = require('./getValidGraphQLTag');
 const getValidRelayQLTag = require('./getValidRelayQLTag');
 const invariant = require('./invariant');
 
+let RelayConfig;
+try {
+  // eslint-disable-next-line no-eval
+  RelayConfig = eval('require')('relay-config');
```

Comment:
i know that we were already using it, but why the `eval('require')('foo')` pattern? is this because the bundler can't find it or something?

Reply:
I think this is to avoid webpack messing `require` calls

Reply:
That’s half of it, yeah.

1. For the single webpack bundle, to avoid webpack from failing to build (as it can’t find user project files at build time), we can’t refer to the `require` symbol. Normally with web pack you could use [`__webpack_require__`](https://webpack.js.org/api/module-variables#__webpack_require__-webpack-specific) instead…
1.  However, as we also build commonjs modules, we can’t use a webpack specific API and have to use plain `require`.

This seemed to be a pattern people use to avoid that situation.